packages/sdk-node/test/methods.spec.ts

```diff
@@ -754,7 +754,7 @@ describe('LookerNodeSDK', () => {
           const query = await sdk.ok(sdk.create_query(request))
           expect(query).toBeDefined()
           expect(query.id).toBeDefined()
-          expect(query.id).toBeGreaterThan(0)
+          expect(Number(query.id)).toBeGreaterThan(0)
```

Comment:
hm, now that IDs really are strings, maybe

```suggestion
          expect(query.id.length).toBeGreaterThan(0)
```

Reply:
TypeScript didn't like that so I'm using a `String()` cast